### ISSUE TITLE
Image field not showing preview/file name on upload

### ISSUE THREAD

**Issue by JElgar** [unfoldadmin/django-unfold#697]
**What version of Unfold are you using?**

0.36.0

**What version of Django are you using?**

For example: 5.1

**What browser are you using?**

Chrome

**Did you checked changelog/commit history, if the bug is not already fixed?**

Yes

**Did you searched other issues, if the bug is not already fixed?**

Yes

**Repository with reproduced bug**

I haven't done this yet, can do if it will help reproduce

**Describe your issue**

Repro:
1. Open a form in the admin panel with an image filed
2. Upload an image to that field

The text does not change (it still says choose file to upload) and there is no preview.
In the console there is an error

```
app.js:122 Uncaught TypeError: Cannot read properties of null (reading 'setAttribute')
    at HTMLInputElement.<anonymous> (app.js:122:19)
```

**Comment by lukasvinclav** [comment]
Is it possible to reproduce the issue here?

https://demo.unfoldadmin.com/admin/formula/driver/56/change/

**Comment by JElgar** [comment]
No... Seems to work there

**Comment by JElgar** [comment]
I had a quick walk through and looks like this code (one my setup)
https://github.com/unfoldadmin/django-unfold/blob/421ee23dc230a81c818c618c8d05fb0b3aee2bef/src/unfold/static/unfold/js/app.js#L127
Is one .parentNode too short

**Comment by lukasvinclav** [comment]
I will check later this issue together with one other related to image field as well. Thanks for report.

**Comment by lukasvinclav** [comment]
Thanks for report. Fixed in #716
